shell/platform/darwin/macos/framework/Source/AccessibilityBridgeMacDelegate.mm

```diff
@@ -265,9 +271,10 @@
     case ui::AXEventGenerator::Event::CHILDREN_CHANGED: {
       // NSAccessibilityCreatedNotification seems to be the only way to let
       // Voiceover pick up layout changes.
+      NSCAssert(flutter_engine_.viewController, @"The viewController must not be nil");
       events.push_back({
           .name = NSAccessibilityCreatedNotification,
-          .target = [NSApp mainWindow],
+          .target = flutter_engine_.viewController.view.window,
```

Comment:
You'll also need to check that the window exists; someone could make a FlutterView and start running it's engine before adding it to a window. If it's nil, you can skip a11y since it's not displayed.

Reply:
oh right, I will fix it. it is hard to imagine how developer would play around with the flutter engine.